thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/datalayer/pojo/DetectionConfigBean.java

```diff
@@ -44,6 +43,8 @@
   Map<String, Object> componentSpecs;
   long lastTuningTimestamp;
   List<String> owners;
+  boolean isDataAvailabilitySchedule;
```

Comment:
isEventBasedTriggering

Reply:
I will prefer the more specific name of the flag, because we might have other triggers in the future.